doc/flatpak-install.xml

```diff
@@ -33,7 +33,7 @@
             <cmdsynopsis>
                 <command>flatpak install</command>
                 <arg choice="opt" rep="repeat">OPTION</arg>
-                <arg choice="plain">REMOTE-NAME</arg>
+                <arg choice="opt">REMOTE-NAME</arg>
```

Comment:
there is a (preexisting) mismatch here between REMOTE-NAME in the display and REMOTE in the paragraph. We should fix that at the same time, maybe ?